ocpkg

```diff
@@ -630,6 +636,77 @@ rm -rf master.tar.gz cogutil-master/
 cd $CURRENT_DIR
 }
 
+# Install notebooks
+install_notebooks(){
+MESSAGE="Installing notebooks...." ; message
+
+
+#####INSTALLING pip3 AND jupyter notebook
+cd
+sudo apt-get install python3-pip
+python3 -m pip install --upgrade pip
+python3 -m pip install ipython jupyter
+
+#####INSTALLING GUILE KERNEL#####
+#check if site folder exits
+if [ ! -d "/usr/local/share/guile/site" ]
+then
+    cd /usr/local/share/guile/
+    sudo mkdir site
+    cd
+fi
+
+#Install ZeroMQ library
+##Clean up remenants of previous installations##
+rm -rf zeromq-4.2.1*
+wget https://github.com/zeromq/libzmq/releases/download/v4.2.1/zeromq-4.2.1.tar.gz
+tar xvf zeromq-4.2.1.tar.gz
+cd zeromq-4.2.1/
+./configure
+make
+sudo make install
+cd
+
+#Install guile-json library
+##Clean up remenants of previous installations##
+rm -rf guile-json-0.6.0*
+wget http://download.savannah.gnu.org/releases/guile-json/guile-json-0.6.0.tar.gz
+tar xvf guile-json-0.6.0.tar.gz
+cd guile-json-0.6.0
+./configure --prefix=/usr/local/
+make
+sudo make install
+cd
+
+cd `guile -c "(display (%global-site-dir))"`
+#Place guile-simple-zmq library to the guile library folder
+sudo rm -rf simple-zmq.scm*
+sudo wget https://raw.githubusercontent.com/jerry40/guile-simple-zmq/master/src/simple-zmq.scm
+cd
+
+#Kernel setup
+if [ ! -d "/home/$USER/.local/share/jupyter/kernels/guile" ]
+then
+    sudo mkdir /home/$USER/.local/share/jupyter/kernels/guile
```

Comment:
why install it in the user's home directory, instead of /usr/local/share/jupyter/kernels?

Reply:
Only a matter of preference, as you can see in the "Kernel Setup" section in https://github.com/jerry40/guile-kernel both are possible.

Reply:
I think it would be best if installed to /usr/local/share/... as it would make it easier when people share the same machine or during `docker run -u $(id -u):$(id -g)`